lib/basedriver/driver.js

```diff
@@ -88,9 +87,16 @@ class BaseDriver extends Protocol {
     // allow subclasses to have internal drivers
     this.managedDrivers = [];
 
-    // store event timings
-    this._eventHistory = {
-      commands: [] // commands get a special place
+    this._meta = {
+      _eventHistory: {
```

Comment:
given that `_meta` is already "private", its subfields don't really need to be (i.e., we can get rid of `_` here I think)

Reply:
done